server/stack.yaml

```diff
@@ -1 +1,6 @@
-resolver: lts-11.20
+resolver: lts-12.6
+
+extra-deps:
+  - prometheus-client-1.0.0
```

Comment:
So the [`prometheus`](https://github.com/bitnomial/prometheus) package is now included in Stackage after I asked for it to be included, but we use a different Prometheus library here? We do use `prometheus` in one of our other projects. Maybe we should ask the author of `prometheus-client` to include it in Stackage too then? Would be nice if we can reduce the number of extra deps.

Reply:
I just checked and we use the `prometheus-client` library in WM and SHM, and we use the `prometheus` library in JM. I don't know the pros and cons of either (the former has 36 GH stars and the latter 19 fwiw).
I'll just leave it like it is for now, since it's working well enough, but we can discuss later to unify on one of those libraries.